src/ZendCodingStandard/ruleset.xml

```diff
@@ -201,232 +70,823 @@
+
+    <!-- 6.2. Binary operators -->
+
+    <!-- All binary arithmetic, comparison, assignment, bitwise, logical,
+         string, and type operators MUST be preceded and followed by at least
+         one space. -->
+    <!-- checked by PSR12.Operators.OperatorSpacing -->
+
+    <!-- 6.3. Ternary operators -->
+
+    <!-- The conditional operator, also known simply as the ternary operator,
+         MUST be preceded and followed by at least one space around both the ?
+         and : characters. -->
+    <!-- When the middle operand of the conditional operator is omitted, the
+         operator MUST follow the same style rules as other binary comparison
+         operators. -->
+    <!-- checked by PSR12.Operators.OperatorSpacing -->
+
+    <!-- 7. Closures -->
+
+    <!-- Closures MUST be declared with a space after the function keyword, and
+         a space before and after the use keyword. -->
+    <!-- The opening brace MUST go on the same line, and the closing brace MUST
+         go on the next line following the body. -->
+    <!-- There MUST NOT be a space after the opening parenthesis of the argument
+         list or variable list, and there MUST NOT be a space before the closing
+         parenthesis of the argument list or variable list. -->
+    <!-- In the argument list and variable list, there MUST NOT be a space
+         before each comma, and there MUST be one space after each comma. -->
+    <!-- Closure arguments with default values MUST go at the end of the
+         argument list. -->
+    <!-- Argument lists and variable lists MAY be split across multiple lines,
+         where each subsequent line is indented once. When doing so, the first
+         item in the list MUST be on the next line, and there MUST be only one
+         argument or variable per line. -->
+    <!-- When the ending list (whether of arguments or variables) is split
+         across multiple lines, the closing parenthesis and opening brace MUST
+         be placed together on their own line with one space between them. -->
+    <!-- checked by Squiz.Functions.MultiLineFunctionDeclaration -->
+
+    <!-- If a return type is present, it MUST follow the same rules as with
+         normal functions and methods; if the use keyword is present, the colon
+         MUST follow the use list closing parentheses with no spaces between
+         the two characters. -->
+    <!-- checked by PSR12.Functions.ReturnTypeDeclaration -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- Inherited variables passed via `use` MUST be used in closures. -->
+    <rule ref="SlevomatCodingStandard.Functions.UnusedInheritedVariablePassedToClosure"/>
+
+    <!-- 8. Anonymous Classes -->
+
+    <!-- Anonymous Classes MUST follow the same guidelines and principles as
+         closures in the above section. -->
+    <!-- The opening brace MAY be on the same line as the class keyword so long
+         as the list of implements interfaces does not wrap. If the list of
```

Comment:
```suggestion
         as the list of implemented interfaces does not wrap. If the list of
```

Reply:
The `implements` keyword was meant here. I've added back-ticks to make that clear. Official text from PSR-12:

> The opening brace MAY be on the same line as the `class` keyword so long as
the list of `implements` interfaces does not wrap. If the list of interfaces
wraps, the brace MUST be placed on the line immediately following the last
interface.